lib/node_modules/@stdlib/string/len/test/test.js

```diff
@@ -0,0 +1,269 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var tape = require( 'tape' );
+var readFileSync = require( '@stdlib/fs/read-file' ).sync;
+var len = require( './../lib' );
+var CR = require( './../lib/constants.js' ).CR;
+var LF = require( './../lib/constants.js' ).LF;
+var Control = require( './../lib/constants.js' ).Control;
+var Extend = require( './../lib/constants.js' ).Extend;
+var RegionalIndicator = require( './../lib/constants.js' ).Regional_Indicator;
+var SpacingMark = require( './../lib/constants.js' ).SpacingMark;
+var L = require( './../lib/constants.js' ).L;
+var V = require( './../lib/constants.js' ).V;
+var T = require( './../lib/constants.js' ).T;
+var LV = require( './../lib/constants.js' ).LV;
+var LVT = require( './../lib/constants.js' ).LVT;
+var Prepend = require( './../lib/constants.js' ).Prepend;
+var ZWJ = require( './../lib/constants.js' ).ZWJ;
+var ExtendedPictographic = require( './../lib/constants.js' ).Extended_Pictographic;
+var Other = require( './../lib/constants.js' ).Other;
+var codePointAt = require( './../lib/code-point-at.js' );
+var getEmojiProperty = require( './../lib/get-emoji-property.js' );
+var getGraphemeBreakProperty = require( './../lib/get-grapheme-break-property' );
+var isSurrogate = require( './../lib/is-surrogate.js' );
+var nextBreak = require( './../lib/next-break.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof len, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the codePointAt function returns code point from string at position', function test( t ) {
```

Comment:
For tests testing the behavior of utility (helper) functions, you can move to separate test files. Each separate test file would be named after the utility file in question. For example, here, we'd have `test.code_point_at.js` which would include only the tests for `codePointAt`.